t/test.lua

```diff
@@ -196,6 +196,10 @@ function issue_120(request)
   return request
 end
 
+function skipping_test(request)
+    return require 'json'.encode(request)
```

Comment:
Nit: it is already required, at top of the file.

Reply:
Done.